main.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/spf13/viper"
 )
 
-const atlantisVersion = "0.18.5"
```

Comment:
the version number in here is wrong, will catch it in next week release.

Reply:
The kustmoize deployment also got impacted, cutting a patch release now. (#2122 )

Reply:
@jvrplmlmn should be handled properly with 0.19.1 release. 🎉